### DISCUSSION TITLE
west update failed to install on network drive, SID error

### DISCUSSION THREAD

**Discussion by Yachter** [zephyrproject-rtos/zephyr#50237] [Q&A] [answered]
=== updating zscilib (modules/lib/zscilib):
--- zscilib: initializing
warning: re-init: ignored --initial-branch=init_placeholder
Reinitialized existing Git repository in //server1/alex/Zephyr/zephyrproject1/modules/lib/zscilib/.git/
fatal: detected dubious ownership in repository at '//server1/alex/Zephyr/zephyrproject1/modules/lib/zscilib'
'//server1/alex/Zephyr/zephyrproject1/modules/lib/zscilib' is owned by:
        'S-1-22-1-556329679'
but the current user is:
        'S-1-5-21-2855892214-3375276786-3802302965-129679'
To add an exception for this directory, call:

        git config --global --add safe.directory '%(prefix)///server1/alex/Zephyr/zephyrproject1/modules/lib/zscilib'
ERROR: update failed for multiple projects; see above

m:\Zephyr\zephyrproject1>

**Comment by cameled** [comment, marked as answer]
`west update` is a simple `git` helper. This issue is related to proper git access permission. please refer https://ittutoria.net/fatal-detected-dubious-ownership-in-repository-what-do-we-need-to-do-to-fix-it/

More detail descriptions for `west update`: https://docs.zephyrproject.org/latest/develop/west/built-in.html#west-update
